Wikipedia/Code/ArticleViewController.swift

```diff
@@ -324,6 +328,9 @@ class ArticleViewController: ViewController, HintPresenting {
     
     override func viewDidAppear(_ animated: Bool) {
         super.viewDidAppear(animated)
+
+        /// When jumping back to an article via long pressing back button (on iOS 14 or above), W button disappears. Couldn't find cause. It disappears between `viewWillAppear` and `viewDidAppear`, as setting this on the `viewWillAppear`doesn't fix the problem. If we can find source of this bad behavior, we can remove this next line.
+        setupWButton()
```

Comment:
Yeah, I think this is related to the standard animation that occurs, I was able to repro with basic back button tap by returning true in `shouldPop` and commenting out the `delegate?.navigationController?.popViewController(animated: true)` line. I think this fix will work though, and it's good that it only happens on the context menu pop.